src/test/java/org/kcctl/command/GetConnectorsCommandTest.java

```diff
@@ -49,6 +51,10 @@ public void should_print_empty_connectors_() {
     @Test
     public void should_print_registered_connectors() {
         registerTestConnectors("test1", "test2");
+        for (String connector : List.of("test1", "test2")) {
+            kafkaConnect.ensureConnectorState(connector, Connector.State.RUNNING);
+            kafkaConnect.ensureConnectorTaskState(connector, 0, Connector.State.RUNNING);
```

Comment:
This test failed in a [prior CI run](https://github.com/kcctl/kcctl/actions/runs/5705038378/job/15459252836). In rare cases, we'd successfully submit the connector to the Connect cluster, and then before tasks for it could be brought up, run the `get connectors` command.

The calls to `ensureConnectorState` and (more importantly) `ensureConnectorTaskState` here ensure that the connector and its task have actually started before we examine the cluster using the CLI.

Reply:
Nice find!